docs/config.ini

```diff
@@ -1,13 +1,13 @@
 # below are the server-wide settings for documentation
 [help_settings]
-url_base = http://edx.readthedocs.org/projects/edx-partner-course-staff
-version = latest
+url_base = http://edx.readthedocs.org/projects/open-edx-ca
+version = dogwood
 
 
 # below are the pdf settings for the pdf file
 [pdf_settings]
-pdf_base = https://media.readthedocs.org/pdf/edx-partner-course-staff
-pdf_file = edx-partner-course-staff.pdf
+pdf_base = https://media.readthedocs.org/pdf/open-edx-ca
```

Comment:
The url is made of three pieces: `{pdf_base}/{version}/{pdf_file}`, so it will end up going to that URL, yes.